ssn/chapters/Common.html

```diff
@@ -1903,7 +1956,7 @@ <h5><a href="#SOSAphenomenonTime">sosa:phenomenonTime</a></h5>
                 <br>
                 <br>
                 In the case of <a href="#SOSAActuation">sosa:Actuation</a>, <a
-                    href="#SOSAphenomenonTime">sosa:phenomenonTime</a> should indicate the time that the <a
+                    href="#SOSAphenomenonTime">sosa:phenomenonTime</a> SHOULD indicate the time that the <a
```

Comment:
"the time at which the sosa:Actuator was active"

Reply:
Yes